backend/docker.go

```diff
@@ -28,6 +29,8 @@ var (
 		"CMD":             "command (CMD) to run when creating containers (default \"/sbin/init\")",
 		"MEMORY":          "memory to allocate to each container (0 disables allocation, default \"4G\")",
 		"CPUS":            "cpu count to allocate to each container (0 disables allocation, default 2)",
+		"CPU_SET_SIZE":    "size of available cpu set (default detected locally via runtime.NumCPU)",
+		"EXEC":            "run build script via `docker exec` instead of over ssh (default true)",
```

Comment:
shall we default this to false for now, until we've fully tested it? just to make sure it doesn't automatically get enabled by someone using it (e.g. enterprise customers)